projects/plugins/migration/src/js/components/migration/index.tsx

```diff
@@ -130,7 +130,7 @@ export function Migration( props: Props ) {
 					isSecondary={ true }
 					target={ '_blank' }
 					href={ getRedirectUrl(
-						'https://wordpress.com/support/import/import-an-entire-wordpress-site/'
+						'https://wordpress.com/support/import-using-wordpress-migration/'
```

Comment:
I would recommend using a Jetpack Redirect unique slug in here, so you can update that URL without having to release the plugin later on. You can read more about how to do that here:
PCYsg-pY7-p2

Reply:
Thanks for the suggestion @jeherve ! In this case I've updated all the static link in the plugin with the slug 👍